diff --git a/ios/AppDelegates/EXReactRootViewFactory.mm b/ios/AppDelegates/EXReactRootViewFactory.mm
index 33e26db44265dca97d4d4c0e74c15cb377480a6b..1bc361f7a1ce30fd975d77cb12dfb68394c8dfdc 100644
--- a/ios/AppDelegates/EXReactRootViewFactory.mm
+++ b/ios/AppDelegates/EXReactRootViewFactory.mm
@@ -52,7 +52,7 @@
                  launchOptions:(nullable NSDictionary *)launchOptions
 {
   if (self.reactDelegate != nil) {
-    return [self.reactDelegate createReactRootViewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions];
+    return [((id<EXReactDelegateProtocol>)self.reactDelegate) createReactRootViewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions];
   }
   return [super viewWithModuleName:moduleName initialProperties:initialProperties launchOptions:launchOptions];
 }
